flexible_event_config.md

```diff
@@ -66,33 +168,38 @@ We will add two optional parameters to the JSON in `Attribution-Reporting-Regist
     //
     // If omitted, then represents a trivial mapping
     // [1, 2, ... , MAX_INT]
-    // With MAX_INT being the maximum possible int value defined by the browser.
+    // With MAX_INT being the maximum int value defined by the browser.
     "summary_buckets": [<bucket start>, ...]
   }, {
     // Next trigger_spec
   } ...],
 
-  // Optional. This is a global parameter that acts across all trigger specs
+  // Part of phase 1. Optional. This is a global parameter that acts across all trigger specs
   // for the lifetime of this source. It restricts the total number of event-level reports
-  // that this source can generate. A report is generated for every "bucket increment",
-  // so this parameter also restricts the total number of bucket increments as well.
+  // that this source can generate. This parameter also restricts the total number of bucket increments as well by limiting the number of conversions per source.
   // After this maximum is hit, the source is no longer capable of producing any new
   // data. The use of priority in the trigger attribution algorithm in the case of
   // multiple attributable triggers remains unchanged.
   // Defaults to 3 for navigation sources and 1 for event sources.
   "max_event_level_reports": <int>
+  // Part of phase 1. Optional. Represents a series of time windows, starting at <start time>.
```

Comment:
We can probably just say:

// See description in phase 1.

As this is a 1:1 copy above the prose above.

Reply:
updated code comments